doc/release/upcoming_changes/15867.deprecation.rst

```diff
@@ -0,0 +1,4 @@
+``numpy.ndarray.tostring()`` is deprecated in favor of ``tobytes()``
+--------------------------------------------------------------------
+`~numpy.ndarray.tobytes` has existed since the 1.9 release, but until this release `~numpy.ndarray.tostring` emitted no warning.
+The change to emit a warning brings NumPy in line with the builtin `array.array` methods of the same name.
```

Comment:
should be reflowed to 80 columns

Reply:
```suggestion
``numpy.ndarray.tostring()`` is deprecated in favor of ``tobytes()``
--------------------------------------------------------------------
`~numpy.ndarray.tobytes` has existed since the 1.9 release, but until this
release `~numpy.ndarray.tostring` emitted no warning. The change to emit a
warning brings NumPy in line with the builtin `array.array` methods of the
same name.
```